lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1402,7 +1402,7 @@ bool SPIRVToLLVM::postProcessOCLBuiltinWithArrayArguments(
           auto Zero =
               ConstantInt::getNullValue(Type::getInt32Ty(T->getContext()));
           Value *Index[] = {Zero, Zero};
-          I = GetElementPtrInst::CreateInBounds(Alloca, Index, "", CI);
+          I = GetElementPtrInst::CreateInBounds(nullptr, Alloca, Index, "", CI);
```

Comment:
We have the desired type already in `T`:
```suggestion
          I = GetElementPtrInst::CreateInBounds(T, Alloca, Index, "", CI);
```